docs/FAQ.md

```diff
@@ -17,6 +17,10 @@ Please change the compiler version in Solidity Plugin to the newer one or use an
 
 **A:**  Try a different browser or a newer solidity compiler version.
 
+**Q:** How to verify a contract that imports other contracts?
+
+**A:**  Use the flattener plugin.
```

Comment:
perhaps should we explain what it does and the reason why we should use the flattener plugin